Gruntfile.js

```diff
@@ -98,33 +95,19 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-file-creator');
     grunt.loadNpmTasks('grunt-webpack');
 
-    // This task should be run once manually at install time.
-    grunt.registerTask('setup', 'Initial install/setup tasks', function () {
-        // If the local config file doesn't exist, we make it
-        var confDir = grunt.config.get('girderDir') + '/conf';
-        if (!fs.existsSync(confDir + '/girder.local.cfg')) {
-            fs.writeFileSync(
-                confDir + '/girder.local.cfg',
-                fs.readFileSync(confDir + '/girder.dist.cfg')
-            );
-            console.log('Created local config file.');
-        }
-    });
-
     /**
-     * Load `default` and `init` targets by topologically sorting the
-     * tasks given by keys the config object.  As in:
+     * Load `default` target by topologically sorting the tasks given by keys the config object.
+     * As in:
      * {
-     *   'init': {
+     *   'default': {
      *     'copy:a': {}
      *     'uglify:a': {
      *       'dependencies': ['copy:a']
      *     }
      *   }
      * }
      *
-     * The init task will run `copy:a` followed by `uglify:a`.
+     * The 'default' task will run `copy:a` followed by `uglify:a`.
      */
-    grunt.registerTask('init', sortTasks(grunt.config.get('init')));
```

Comment:
I don't know of any examples of this, but it's possible there might be plugins that rely on the init task.  What do you think about replacing this task with a deprecation warning?  I think you could also merge the `init` object with the `default` object before running the next line.